pallets/evercity/src/tests.rs

```diff
@@ -2367,11 +2400,11 @@ fn bond_impact_report_missing_data() {
         bond_activate(bondid1, ACCOUNT1, bond.clone());
         bond_activate(bondid2, ACCOUNT2, bond.clone());
 
-        for period in &[0, 1, 3, 5, 7, 9] {
+        for &period in &[0, 1, 3, 5, 7, 9] {
             assert_ok!(Evercity::set_impact_data(
```

Comment:
add separate test when NO set_impact_data() is called, when bod was "forgotten" and nobody sends data to it. Test is needed to check conditions where all periods are "missed"